embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/arch/schema/EmbType.kt

```diff
@@ -58,6 +58,40 @@ internal sealed class EmbType(type: String, subtype: String?) : TelemetryType {
         }
 
         internal object Exit : System("exit")
+
+        internal object Crash : System("android.crash") {
```

Comment:
what about `emb.android.crash.count` and `emb.android.threads`

Reply:
These are defined in `EmbraceAttributeKeys` because they don't below to a particular crash type - `emb.android.crash_number` is actually metadata for the session that we happen to stick on each crash. The same count is shared amongst all types of crashes.

For threads, I can see this being useful outside of crashes, so I extracted it out of the crash domain.